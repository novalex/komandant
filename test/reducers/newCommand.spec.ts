import { AnyAction } from 'redux';

import newCommandReducer, {
	initialState,
} from '../../app/features/new-command/slice';

describe('reducers', () => {
	describe('counter', () => {
		it('should handle initial state', () => {
			expect(
				newCommandReducer(undefined, {} as AnyAction)
			).toMatchSnapshot();
		});

		it('should handle unknown action type', () => {
			expect(
				newCommandReducer(initialState, { type: 'unknown' })
			).toMatchSnapshot();
		});
	});
});
