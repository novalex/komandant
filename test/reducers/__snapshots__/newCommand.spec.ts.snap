// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers counter should handle initial state 1`] = `
Object {
  "fields": Array [],
}
`;

exports[`reducers counter should handle unknown action type 1`] = `
Object {
  "fields": Array [],
}
`;
